geomet_climate/tileindex.py

```diff
@@ -201,13 +203,19 @@ def create_dataset(layer_info, input_dir, output_dir):
                                 layer_info['filename'])
 
         LOGGER.info('Generating GPKG')
+        nodata_check = False
         for key in file_time:
             LOGGER.debug('Adding feature to layer')
             if not key.endswith('.tif'):
+                if nodata_check is False:
+                    netcdf_ds = gdal.Open(filename)
```

Comment:
Can you close the file i.e. `netcdf_ds = None`.

Reply:
done